scripts/globals/status.lua

```diff
@@ -1250,6 +1250,7 @@ MOD_FORCE_LIGHT_DWBONUS       = 537 -- Set to 1 to force light day/weather spell
 MOD_FORCE_DARK_DWBONUS        = 538 -- Set to 1 to force dark day/weather spell bonus/penalty. Do not have it total more than 1.
 MOD_STONESKIN_BONUS_HP        = 539 -- Bonus "HP" granted to Stoneskin spell.
 MOD_ENHANCES_ELEMENTAL_SIPHON = 540 -- Bonus Base MP added to Elemental Siphon skill.
+MOD_BP_DELAY_II               = 541  -- Blood Pact Delay Reduction II
```

Comment:
Would be nice if this could be renamed to `MOB_BLOODPACK_DELAY_II`, yeah it's really long but no one will have the guess the meaning.